js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/EvaluationConditionalLabel.tsx

```diff
@@ -4,15 +4,139 @@ import {
+  }
+
+  const [triggerCondition, _operator, resetCondition] = expandedLabel;
+  const triggerLabel = triggerCondition?.slice(1, -1) || ''; // remove parentheses
+  const resetLabel = resetCondition?.slice(1, -1) || ''; // remove parentheses
+  const triggerTime = timestampToString({
+    timestamp: {unix: sinceMetadata.triggerTimestamp},
+    locale,
+    timezone: userTimezone,
+    timeFormat: DEFAULT_TIME_FORMAT,
+    hourCycle: userHourCycle,
+  });
+  const resetTime = timestampToString({
+    timestamp: {unix: sinceMetadata.resetTimestamp},
+    locale,
+    timezone: userTimezone,
+    timeFormat: DEFAULT_TIME_FORMAT,
+    hourCycle: userHourCycle,
+  });
+
+  const assetKey =
+    entityKey && entityKey.__typename === 'AssetCheckhandle' ? entityKey.assetKey : entityKey;
+  const checkName =
+    entityKey && entityKey.__typename === 'AssetCheckhandle' ? entityKey.name : undefined;
+
+  return (
+    <Box flex={{direction: 'row', gap: 8, wrap: 'wrap', alignItems: 'center'}}>
+      <Tooltip content={<TooltipContent text={triggerLabel} />} placement="top">
+        <Operand>{triggerLabel}</Operand>
+      </Tooltip>
+      <EvaluationSinceMetadata
+        assetKey={assetKey}
+        checkName={checkName}
+        detailLabel={`${triggerLabel} was last True at ${triggerTime}`}
+        evaluationId={sinceMetadata.triggerEvaluationId || 0}
+        timestamp={sinceMetadata.triggerTimestamp || 0}
+        pushHistory={pushHistory}
+      />
+      <Operator>SINCE</Operator>
+      <Tooltip content={<TooltipContent text={resetLabel} />} placement="top">
+        <Operand>{resetLabel}</Operand>
+      </Tooltip>
+      <EvaluationSinceMetadata
+        assetKey={assetKey}
+        checkName={checkName}
+        detailLabel={`${resetLabel} last occurred ${resetTime}`}
+        evaluationId={sinceMetadata.resetEvaluationId || 0}
+        timestamp={sinceMetadata.resetTimestamp || 0}
+        pushHistory={pushHistory}
+      />
+    </Box>
+  );
+};
+
+export const EvaluationSinceMetadata = ({
+  assetKey,
+  checkName,
+  detailLabel,
+  evaluationId,
+  pushHistory,
+}: {
+  assetKey: {path: string[]};
+  checkName?: string;
+  detailLabel: string;
+  evaluationId: number;
+  timestamp: number;
+  pushHistory?: (item: EvaluationHistoryStackItem) => void;
+}) => {
+  if (!pushHistory) {
+    return (
+      <Tooltip content={detailLabel}>
+        <Icon name="info" color={Colors.accentGray()} style={{verticalAlign: 'middle'}} />
+      </Tooltip>
+    );
+  }
+  return (
+    <Tooltip content={detailLabel}>
+      <a
+        onClick={(e) => {
+          e?.stopPropagation();
+          pushHistory({
+            assetKeyPath: assetKey.path,
+            assetCheckName: checkName,
+            evaluationID: String(evaluationId),
+          });
+        }}
+      >
+        <Icon name="link" color={Colors.accentGray()} style={{verticalAlign: 'middle'}} />
+      </a>
```

Comment:
If you're not using an href/to, use an `UnstyledButton`​ or `ButtonLink`​ or something similar, instead of an `a`​. (And ideally you won't need a `stopPropagation`​.)  
  
Is it definitely better to use pushing to history instead of a `<Link to={...}>`​?